packages/react-components/src/components/styles/AttachmentCard.styles.ts

```diff
@@ -34,7 +34,6 @@ export const useAttachmentCardStyles = makeStyles({
   },
   content: {
     '> div': {
-      width: '100%',
```

Comment:
Should this have been removed?

Reply:
Yes, after FluentUI dependencies update, it caused icon, text and action menu in atttachment card to be the same size.

Reply:
Refactored styles a little bit to simplify the UI